src/models/methods/baseMethod.ts

```diff
@@ -0,0 +1,24 @@
+export interface BaseRequest {
+    id: number | string
+    command: string
+    api_version?: number
+}
+
+interface Warning {
+    id: number
+    message: string
+    details?: {[key: string]: string}
+}
+
+export interface BaseResponse {
+    id: number | string
+    status: string
```

Comment:
Can status be anything other than `success` or `error`?

Reply:
I'm not entirely sure, which is why I left it as `string` - maybe @intelliot knows

Reply:
I don't know. I think this is fine as-is, or `"success" | string` if you want to document the fact that 'success' is the happy path value